### PR TITLE
Fix build and cleanup images

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 PYTHON ?= python
 PIP ?= pip
+endif 
+
+ifneq ($(RHEL5),)
+TEST_INSTALL_CMD=
+endif
+
+ifneq ($(RHEL6),) # python 2.6 busted for flake8
+TEST_INSTALL_CMD=$(PIP) install -r test-requirements-py26.txt --user
```
Comment (evgeni) [inline Makefile]: you don't need a special `requirements.txt` for EL6, you can just write:

```
flake8; python_version >= '2.7'
mock
unittest2
```

[pip docs](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers)
[pep 508](https://www.python.org/dev/peps/pep-0508/#environment-markers)

Comment (jturel) [reply Makefile]: Wow nice, updated!

File: images/Dockerfile.el7 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM centos:7
 
-COPY containers/assets/jortel-gofer-epel.repo /etc/yum.repos.d/jortel-gofer-epel.repo
+RUN yum install -y http://yum.theforeman.org/client/1.21/el7/x86_64/foreman-client-release.rpm && yum clean all
```
Comment (evgeni) [inline images/Dockerfile.el7]:
```suggestion
RUN yum install -y https://yum.theforeman.org/client/1.21/el7/x86_64/foreman-client-release.rpm && yum clean all
```

File: images/Dockerfile.el6 [hunk 3]
```diff
@@ -1,8 +1,7 @@
 FROM centos:6
 
 RUN yum install -y wget epel-release && yum clean all
-RUN yum install -y https://fedorapeople.org/groups/katello/releases/yum/3.4/client/el6/x86_64/katello-client-repos-latest.rpm && yum clean all
-RUN wget -O /etc/yum.repos.d/epel-subscriptions-manager.repo http://repos.fedorapeople.org/repos/candlepin/subscription-manager/epel-subscription-manager.repo
-RUN yum install make gofer python-pulp-agent-lib subscription-manager python-pip -y && yum clean all
+RUN yum install -y http://yum.theforeman.org/client/1.21/el6/x86_64/foreman-client-release.rpm && yum clean all
```
Comment (evgeni) [inline images/Dockerfile.el6]:
```suggestion
RUN yum install -y https://yum.theforeman.org/client/1.21/el6/x86_64/foreman-client-release.rpm && yum clean all
```

**Comment by jturel** [comment]
Thanks @evgeni !

**Pull request merged**